src/js/edu-benefits/definitions/serviceBefore1977.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import commonDefinitions from 'vets-json-schema/dist/definitions.json';
+
+export const schema = commonDefinitions.serviceBefore1977;
+
+export const uiSchema = {
+  married: {
+    'ui:title': 'Are you currently married?',
+    'ui:widget': 'yesNo'
+  },
+  haveDependents: {
+    'ui:title': 'Do you have any children who fall into these categories?',
+    'ui:description': () => (
```

Comment:
Just curious why you set this to be a function instead of just setting it to a string of the markup itself?